packages/mdc-floating-label/README.md

```diff
@@ -79,6 +79,7 @@ Mixin | Description
 `mdc-floating-label-shake-keyframes($modifier, $positionY, $positionX, $scale)` | Generates a CSS `@keyframes` at-rule for an invalid label shake. Used in conjunction with the `mdc-floating-label-shake-animation` mixin.
 `mdc-floating-label-shake-animation($modifier)` | Applies shake keyframe animation to label.
 `mdc-floating-label-float-position($positionY, $positionX, $scale)` | Sets position of label when floating.
+`mdc-floating-label-width($width))` | Sets width of label when floating.
 
```

Comment:
Update the description to be "Sets the width of the label" since it actually sets the width of the label in both its floating and non-floating position.

Reply:
Done!